packages/currency/src/currency_types/index.ts

```diff
@@ -7,3 +7,5 @@ export { default as BaseCurrencyType } from './BaseCurrencyType'
 export { default as MobileTokens } from './MobileTokens'
 export { default as SolTokens } from './SolTokens'
 export { default as IotTokens } from './IotTokens'
+import { Ticker } from './BaseCurrencyType'
```

Comment:
do you need to import and then export? or could you combine into an `export ... from` statement?

Reply:
yeah, I don't know why, but that wasn't working for me last night, but it is today. lol.